presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentUp.java

```diff
@@ -82,8 +83,8 @@ public void run()
         @Option(name = "--environment", title = "environment", description = "the name of the environment to start", required = true)
         public String environment;
 
-        @Option(name = "--startup-timeout", title = "environment startup timeout", description = "environment startup timeout")
-        public Duration startupTimeout = Duration.valueOf("10m");
+        @Option(name = "--logs-dir", title = "logs dir", description = "location of the exported logs directory")
```

Comment:
Please rebase onto #4908

`title` should become `paramLabel = `<dir>` and `description` should start with upppercase

Reply:
Sure. Thx @electrum :)